scripts/ci-test.sh

```diff
@@ -24,9 +24,10 @@ cd "${REPO_ROOT}" || exit 1
 # shellcheck source=./hack/ensure-go.sh
 source "${REPO_ROOT}/hack/ensure-go.sh"
 
-make generate test
```

Comment:
I've removed the generate pre-req from here, given that we're already checking it in `pr-verify` presubmit. This reduced the runtime for tests in CI from ~12 minutes to ~3 mins